Cargo.toml

```diff
@@ -2,12 +2,20 @@
 name = "kvproto"
 version = "0.0.1"
 keywords = ["TiKV"]
+edition = "2018"
```

Comment:
Is it OK that dependant project is not written in 2018 edition?

Reply:
Yes, it's fine. The editions are designed so that a 2018 crate can depend on a 2015 crate and a 2015 crate can depend on a 2018 crate.